app/views/pregnancies/_patient_information.html.erb

```diff
@@ -77,7 +77,53 @@
                        options_for_select(referred_by_options,
                                           pregnancy.referred_by),
                                           autocomplete: 'off' %>
-          <%= f.text_field :special_circumstances, autocomplete: 'Off' %>
+          <div>       
+            <%= f.label :special_circumstances, 'Special circumstances' %>   
+            <br />
+            <div>
+              <div class="col-sm-6">
+                <div>
+                  <%= check_box_tag('pregnancy[special_circumstances][]', 'Fetal anomaly', 
+                            @pregnancy.special_circumstances.include?('Fetal anomaly')) %>
+                  <%= label_tag('Fetal anomaly') %>
+                </div>
+                <div>
+                <%= check_box_tag('pregnancy[special_circumstances][]', 'Homelessness', 
+                          @pregnancy.special_circumstances.include?('Homelessness')) %>
+                <%= label_tag('Homelessness') %>
+                </div>
+                <div>
+                  <%= check_box_tag('pregnancy[special_circumstances][]', 'Domestic Violence', 
+                            @pregnancy.special_circumstances.include?('Domestic Violence')) %>  
+                  <%= label_tag('Domestic Violence') %>
+                </div>
+                <div>
+                  <%= check_box_tag('pregnancy[special_circumstances][]', 'Other', 
```

Comment:
I think for an MVP we can put the `other` in the backlog, and just leave it up to CMs to take good notes. I think this is a pretty important chunk and I'd rather have most of this than none of it!